### PR TITLE
Add typing bubble animation

### PR DESCRIPTION
**Pull request by dionyziz**

### CODE REVIEW

**Comment by dionyziz** [comment]
Screencast:

https://www.youtube.com/watch?v=Wml8wNrzONE&feature=youtu.be

**Comment by VitSalis** [comment]
IMO it's animating too fast.

**Comment by dionyziz** [comment]
Doubled the animation time.

File: client/index.css [hunk 1]
```diff
@@ -191,3 +191,44 @@ ul {
     padding: 2px 7px 2px 4px;
     color: transparent;
 }
+div.menu {
```
Comment (gtklocker) [inline client/index.css]: Why is this here?

**Comment by dionyziz** [comment]
@gtklocker PTAL

**Comment by gtklocker** [comment]
LGTM
